app/home/component/Card/Card.tsx

```diff
@@ -0,0 +1,76 @@
+import {
+  DocumentCard,
+  DocumentCardActions,
+  DocumentCardPreview,
+  DocumentCardTitle,
+  ImageFit,
+  Stack,
+  Text,
+  getTheme,
+} from "@fluentui/react";
+
+import { CARD_PREVIEW_IMAGE_SIZE_HEIGHT, CARD_PREVIEW_IMAGE_SIZE_WIDTH } from "../../const";
+
+export type CardType = {
+  title: string;
+  imagePath: string;
+  releaseDate: string;
+  id: number;
+};
```

Comment:
It's a good idea to declare card type here. It seems like `CardPros` doesn't serve anything special, so you can directly use this type for the property without assign this to `CardProps`

Reply:
Assigned CardType as the props type.